src/js/containers/homepage/HomepageContainer.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+  * HomepageContainer.jsx
+  * Created by Emily Gullo 03/15/2017
+  **/
+
+import React from 'react';
+import Landing from 'components/homepage/Landing';
+import TreeMap from 'components/homepage/visualizations/TreeMap';
+import TreeMapIntro from 'components/homepage/TreeMapIntro';
+import LinksSection from 'components/homepage/LinksSection';
+import SearchSection from 'components/homepage/SearchSection';
+import Header from '../../components/sharedComponents/header/Header';
+import Footer from '../../components/sharedComponents/Footer';
+
+export default class HomepageContainer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
Will this data eventually be populated by an API call? Is that why this file is a container and not a component?